lib/flutter_date_picker.dart

```diff
@@ -101,15 +100,15 @@ class DatePickerState extends State<DatePicker> {
 
   @override
   void initState() {
-    _fromYear = widget.fromYear??_date.year - 100;
-    _toYear = widget.toYear??_date.year + 20;
+    int _toYear = widget.toYear??_date.year + 20;
 
     assert(_fromYear <= _initialYear && _initialYear <= _toYear, 
```

Comment:
this throws error when i give 
initialYear: 2019,
fromYear: 1920,
toYear:2020

Reply:
My bad, the assert statement should come after _initialYear instantiation

Reply:
```suggestion
  void initState() {
    int _toYear = widget.toYear??_date.year + 20;
    _fromYear = widget.fromYear??_date.year - 100;
    _initialYear  = widget.initialYear??_date.year;
    
    assert(_fromYear <= _initialYear && _initialYear <= _toYear, 
      "Date Interval Error");
    _selectedYear = _initialYear - _fromYear;
    _selectedMonth = _date.month - 1;
    _selectedDate = _date.day - 1;
    _yearRange = _toYear - _fromYear;
    _dobMonth = monthNames[_date.month - 1];
    super.initState();
} 
```

Reply:
` int _toYear = widget.toYear??_date.year + 20;
    _fromYear = widget.fromYear??_date.year - 100;
    _initialYear  = widget.initialYear??_date.year;
    
    assert(_fromYear <= _initialYear && _initialYear <= _toYear, 
      "Date Interval Error");
    _selectedYear = _initialYear - _fromYear;
    _selectedMonth = _date.month - 1;
    _selectedDate = _date.day - 1;
    _yearRange = _toYear - _fromYear;
    _dobMonth = monthNames[_date.month - 1];
    super.initState();`

Reply:
`dart`
 int _toYear = widget.toYear??_date.year + 20;
    _fromYear = widget.fromYear??_date.year - 100;
    _initialYear  = widget.initialYear??_date.year;
    
    assert(_fromYear <= _initialYear && _initialYear <= _toYear, 
      "Date Interval Error");
    _selectedYear = _initialYear - _fromYear;
    _selectedMonth = _date.month - 1;
    _selectedDate = _date.day - 1;
    _yearRange = _toYear - _fromYear;
    _dobMonth = monthNames[_date.month - 1];
    super.initState();